specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+|                           | Workflow Automation               | Use of software to complete orchestration tasks and activities without the need for human input.                                                                                                       |
+|                           | Other (Integration)               | Integration services that do not fall into one of the defined subcategories.                                                                                                                           |
+| Internet of Things        | Other (Internet of Things)        | All Internet of Things (IoT) services.                                                                                                                                                                 |
+| Management and Governance | Administration                    | Management and maintenance of cloud systems.                                                                                                                                                           |
+|                           | Architecture                      | Planning, design, and construction of cloud systems.                                                                                                                                                   |
+|                           | Compliance                        | Adherance to regulatory standards and industry best practices.                                                                                                                                         |
+|                           | Cost Management                   | Monitoring and controlling expenses of cloud systems.                                                                                                                                                  |
+|                           | Data Governance                   | Management of the availability, usability, integrity, and security of data.                                                                                                                            |
+|                           | Disaster Recovery                 | Plans and procedures that ensure cloud systems can recover quickly from disruptions.                                                                                                                   |
+|                           | Observability                     | Monitoring, logging, and tracing of data to track the performance and health of systems.                                                                                                               |
+|                           | Support                           | Planning, advice, and expertise supplied by cloud providers.                                                                                                                                           |
```

Comment:
```suggestion
|                           | Support                           | Assistance and expertise supplied by providers to enable customers.                                                                                                                                           |
```

Reply:
To enable customers... doing what?  I would prefer to end that sentence with an action or actions customers take.

Reply:
```suggestion
|                           | Support                           | Assistance and expertise supplied by providers.                                                                                                                                           |
```